src/main.ts

```diff
@@ -1,3 +1,5 @@
+import 'tslib';
```

Comment:
this is ugly - do users do this already? why is it needed?

Reply:
no.. this is because tslib is automatically included in the angular bundles but when building from source you have to require it in main to get it in there

Reply:
maybe something in Angular should depend on it. Who actually uses a symbol that requires tslib to be present? It's related to the --noHelpers flag in the tsconfig right?